scripts/build.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
```

Comment:
why is this done over using the package.json files? for new people this adds complexity over using a simple setup that just uses lerna and the package.json files.

Reply:
Running 1 TS compiler is much less CPU effective than 38. I tested both variants and this way it just works more efficient. At the end of the day it is the same command for the developer.

Reply:
Nice to know, thanks!